wait a sec.txt

```diff
@@ -0,0 +1,4 @@
+So you are mass following people on github so that they will follow back, then unfollowing afterwoods to farm a massive follower count?
```

Comment:
i read this before i follow back💀

Reply:
exactly! man this is low key interesting now 😂